packages/core/src/modules/mdoc/MdocDeviceResponse.ts

```diff
@@ -113,7 +148,18 @@ export class MdocDeviceResponse {
 
     const inputDescriptor = this.assertMdocInputDescriptor(options.inputDescriptor)
     const _mdoc = parseIssuerSigned(TypedArrayEncoder.fromBase64(mdoc.base64Url), mdoc.docType)
-    return mdocLimitDisclosureToId({ mdoc: _mdoc, inputDescriptor })
+
+    const disclosure = mdocLimitDisclosureToId(_mdoc, inputDescriptor)
```

Comment:
Id is maybe a bit offputting as `inputDescriptor` ( i thought which ID?). So mabye just `mdocLimitDisclosureToInputDescriptor`?